3rdparty/kde/CMakeLists.txt

```diff
@@ -4,7 +4,7 @@ set(gammaray_kitemmodels_srcs
   krecursivefilterproxymodel.cpp
 )
 
-add_library(gammaray_kitemmodels SHARED ${gammaray_kitemmodels_srcs})
+add_library(gammaray_kitemmodels ${GAMMARAY_LIBRARY_TYPE} ${gammaray_kitemmodels_srcs})
```

Comment:
The proper way would have been to introduce a 'real' static build option to the entire gammaray, especially at least for the probe and it's actually shared only plugins - grep for gammaray_add_plugin to find them.